File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>

int main(){
    int money=-1;
    while (money<0 || money%1>0 ){
        money=get_float("Change owed:")*1000/10;
    }
    int coins[]={25,10,5,1};
    int N=0;
    for(int i=0;money!=0 || i<sizeof(coins)/sizeof(coins[0]);i++){
        N+=money/coins[i];
        money%=coins[i];
    }
    printf("%d\n",N);
    return 0;
}

File: pset6/cash/cash.py
from cs50 import get_float

N=get_float("Change owed:")
while(N<0):
    N=get_float("Change owed:")
N=int(N*100)

coin=[25,10,5,1]
count_=0
for c in coin:
    count_+=N//c
    N%=c
print(count_)
